crates/forge-runner/src/coverage_api.rs

```diff
@@ -0,0 +1,40 @@
+use anyhow::{Context, Result};
+use shared::command::CommandExt;
+use std::process::Stdio;
+use std::{env, fs, path::PathBuf, process::Command};
+
+pub const COVERAGE_DIR: &str = "coverage";
+pub const OUTPUT_FILE_NAME: &str = "coverage.lcov";
+
+pub fn run_coverage(saved_trace_data_paths: &[PathBuf]) -> Result<()> {
```

Comment:
Just to make sure, should it be named `saved_trace_data_paths` and not something like `saved_coverage_data_paths`?

Reply:
These are trace data files saved during the current tests execution. They are used as an input for the `cairo-coverage`.